packages/components/Atoms/Global/scss/settings/_color.scss

```diff
@@ -1,9 +1,28 @@
+// Fallback colors.
+$rpl-colors-core: (
+  'primary': #0052c2,
+  'dark_primary': #023b89,
+  'secondary': #0095ec,
+  'extra_dark_neutral': #011a3c,
+  'dark_neutral': #465870,
+  'dark_neutral_1': #546a7c,
+  'mid_neutral_1': #d7dbe0,
+  'mid_neutral_2': #e8ebee,
+  'light_neutral': #f6f6f9,
+  'danger': #af272f,
+  'warning': #ca4c21,
+  'success': #009ca6,
+  'white': #fff,
+  'black': #000
+);
+
 $rpl-colors: (
   'primary': #0052c2,
```

Comment:
Hi @alan-cole Glad to see it works! Should we then leave this `$rpl-colors` empty? instead of duplicate colors again?

Reply:
Yep. I've now removed the duplicated colors.